src/server/install/index.md

```diff
@@ -7,7 +7,34 @@ Installation guide will help you to install your <CommunityPlatformNameLink /> o
 
 ## Installation System Requirements
 
-We recommend using a dedicated host machine with 8 GB of memory. The requirements for CPU and persistent storage depend largely on the frequency of project updates and the anticipated size of the data you expect to store respectively.
+We recommend using a dedicated host machine with **8 GB** of memory and **4 vCPUS**. The requirements for CPU and persistent storage depend largely on the frequency of project updates and the anticipated size of the data you expect to store respectively.
```

Comment:
something like `t3a.large` equivalent should be enough but surely we can have some buffer

Reply:
`t3a.large` has only 2vCPUs, is this enough to handle generic enterprise deployments?